builder.json

```diff
@@ -7,7 +7,7 @@
         "mvn -P continuous-integration -B compile"
     ],
     "test_steps": [
-        "mvn -B test -DredirectTestOutputToFile=true -DforkCount=0 -DrerunFailingTestsCount=5"
+       "aws-crt-java-test"
```

Comment:
What is this?

Reply:
that's using the python script to run test. https://github.com/awslabs/aws-crt-java/pull/346/files#diff-ce63a05ed0ed2cf8fc220bbf3cf6fc06e4034fae5fe8be8404060bfaf1bf0d98
So, in order to print the log out, we need to continue the command even if the first cmd failed. Get around it by two different `os.system()`.

Reply:
Is there a builder version bump needed?  The script is using underscores, this is using dashes.

Reply:
Nope, the builder will find the script based on the name https://github.com/awslabs/aws-crt-java/pull/346/files#diff-ce63a05ed0ed2cf8fc220bbf3cf6fc06e4034fae5fe8be8404060bfaf1bf0d98R18